src/main/java/com/nbe8101team03/domain/orders/controller/OrderController.java

```diff
@@ -0,0 +1,59 @@
+package com.nbe8101team03.domain.orders.controller;
+
+import com.nbe8101team03.domain.orders.dto.CreateOrderRequest;
+import com.nbe8101team03.domain.orders.dto.OrderResponse;
+import com.nbe8101team03.domain.orders.entity.Order;
+import com.nbe8101team03.domain.orders.service.OrderService;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @PostMapping("/create")
+    public OrderResponse createOrder(@RequestBody CreateOrderRequest req) {
+        return orderService.createOrder(
+                req.email() ,
+                req.address() ,
+                req.zipcode(),
+                req.productId(),
+                req.quantity()
+
+        );
+    }
+    
+//    주문 단건 조회
+    @GetMapping("/{orderId}")
+    public OrderResponse getOrder(@PathVariable Long orderId) {
+        return orderService.getOrder(orderId);
+    }
```

Comment:
응답 형식을 통일하는건 어떨까요? 

```suggestion
    public ResponseEntity<Response<OrderResponse>> getOrder(@PathVariable Long orderId) {
        OrderResponse res = orderService.getOrder(orderId);
        return ResponseEntity.ok(CommonResponse.success(res), "주문 생성을 성공하였습니다.");
    }
```
이와 같은 형식을 다른 엔드포인트(API)에도 적용하는 것이 좋아 보입니다.

---

여담으로, 뭔가 이상해서 제가 작성한 코드를 봤는데 `ResponseEntity` 를 빼놨네요... 이 `ResponseEntity` 로 감싸야 응답 상태 코드를 지정할 수 있습니다. 뭐 200 OK, 201 , 400 같은 친구들이요.

Reply:
피드백반영완료하였습니다